extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -81,6 +98,20 @@ class AmpAccordion extends AMP.BaseElement {
         content.setAttribute('aria-expanded', 'true');
       }
     }, content);
+    // TODO(zhouyx): ask if there's way to get index of currentTarget
+    // element directly without iterating all section.
+    var tempSession_ = '';
```

Comment:
I'd store a structured value like

`{state: [true, false]}`

You may need `JSON.parse` and `JSON.stringify`.